scripts/gen_kobject_list.py

```diff
@@ -202,14 +202,14 @@ def write_kobj_otype_output(fp):
         if dep:
             fp.write("#ifdef %s\n" % dep)
 
-        fp.write('case %s: return "%s";\n' % (kobject_to_enum(kobj), kobj))
+        fp.write('case %s: ret = "%s"; break;\n' % (kobject_to_enum(kobj), kobj))
```

Comment:
Too lazy to check: but the end of this generated switch statement is 100% for sure a "return ret;", right?